stumpy/mmotifs.py

```diff
@@ -0,0 +1,210 @@
+import numpy as np
+import logging
+
+# from .aamp_motifs import aamp_motifs
+from . import core, config, mdl, match
+
+logger = logging.getLogger(__name__)
+
+
+# @core.non_normalized(aamp_motifs)
+def mmotifs(
+    T: np.ndarray,

```

Comment:
Unfortunately, we don't do type annotations in this repo as we've found that they are more distracting for people to read. Can you please remove the `: np.ndarray`?